src/components/SitewideHeader/HamburgerMenu.js

```diff
@@ -96,9 +96,12 @@ const useStyles = createUseStyles(() => ({
   },
 }));
 
-const HamburgerMenu = () => {
+const HamburgerMenu = ({ siteMetadata }) => {
   const classes = useStyles();
 
+  const TWITTER_URL = `https://twitter.com/${siteMetadata.social.twitter}`;
```

Comment:
Why not store the whole url in the siteMetadata instead?

Reply:
This setup (twitter username but not URL in config) comes with Gatsby by default so I kinda just stuck with it. I guess one advantage is that we have the twitter username available if we ever need that in non-linked form. We can count on the Twitter hostname not changing anytime soon so it's basically static config.

With that said, I'm not actually using the metadata version in the code where I reference the username. I will fix that.